_posts/2016-05-11-aws-lambdas.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: post
+name: aws-lambdas 
+title: AWS Lambda for Beginners
+date: 2016-05-11 00:20:00 +00:00
+author: Mashooq Badar 
+image:
+   src: /assets/img/custom/blog/aws-lambda.png
+tags:
+- microservices 
+- devops 
+- aws
+---
```

Comment:
no  mention of AWS Lambda or just Lambda here (former might help) as the light might be mistaken to various other concepts.